books/RayTracingTheNextWeek.html

```diff
@@ -1222,13 +1260,22 @@
 
 </div>
 
-
-Image Textures
----------------
+You may think the result looks a bit odd. Since `checker_texture` is a spatial texture, we're really
+looking at the surface of the sphere cutting through the three-dimensional checker space. There are
+many situations where this is perfect, or at least sufficient. In many other situations, we really
+want to get a consistent effect on the surface of our objects. That approach is covered next.
 
 
 Texture Coordinates for Spheres
 --------------------------------
+Constant-color textures use no coordinates. Solid (or spatial) textures use the coordinates of a
+point in space. Now it's time to make use of the $u,v$ texture coordinates. These coordinates
+specify the location on 2D source image (or in some 2D parameterized space). To get this, we need a
+way to find the $u,v$ coordinates of any point on the surface of a 3D object. This mapping is
+completely arbitrary, but generally you'd like to have cover the entire surface, and scale, orient
```

Comment:
"... you'd like to cover ..."

Reply:
"completely arbitrary, but generally you'd like to cover the entire surface, and be able to scale, orient, and stretch ..."

Feel free to make a judgement call on the oxford comma